cms/djangoapps/contentstore/rest_api/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+Contentstore API URLs.
+"""
+
+from django.urls import include, re_path
```

Comment:
Instead of `re_path`, everywhere we use `url` instead. Can you follow the convention? See example at https://github.com/edx/edx-platform/blob/master/cms/djangoapps/api/urls.py#L12

Reply:
The reason I did not use `url` is because it's not used in the Django docs (e.g. the URL dispatcher docs) at all. The last version of Django I can find that references `django.conf.url` is version [1.11](https://docs.djangoproject.com/en/1.11/topics/http/urls/#naming-url-patterns). And I believe that we are on version 2.2 at this point. The equivalent `url` function in Django 2.2 seems to be located [here](https://docs.djangoproject.com/en/2.2/ref/urls/#url) (the module is django.urls instead of django.conf.urls), and it's actually an alias to `re_path`. The docs also suggest that the `url` alias will be deprecated in a future release. It seems more future proof to use `re_path`. I don't understand the advantage of using `url` if it is functionally equivalent to `re_path`, and `url` will be deprecated in the future. Is there something I'm missing?

Reply:
Aw, that make sense then what you are saying. I rescind my comments on this.